packages/node_modules/@ciscospark/internal-plugin-ediscovery/src/ediscovery.js

```diff
@@ -0,0 +1,520 @@
+/* eslint-disable prefer-template */
+/*!
+ * Copyright (c) 2015-2017 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import {waitForValue, SparkPlugin} from '@ciscospark/spark-core';
+import reportRequest from './report-request';
+import ReportGenerator from './report-generator';
+
+/**
+ * @class Ediscovery is used by compliance officers to run compliance reports
```

Comment:
I would probably rename to `EDiscovery` to be consistent with other names within the repo. This name is a unique though.